decoder/vaapidecoder_mpeg2.cpp

```diff
@@ -289,9 +289,6 @@ YamiStatus VaapiDecoderMPEG2::start(VideoConfigBuffer* buffer)
         m_parser->nextStartCode(m_stream.get(), next_code);
         DEBUG("start Next start_code %x", next_code);
         status = processConfigBuffer();
-        if (status == YAMI_DECODE_FORMAT_CHANGE) {
```

Comment:
Hi @uartie 
Actually, we use start to decode codec data. In some usecases, codec data will send in other channel and it will send multiple times. Change start() to decodeHeader() maybe more reasonable.